ansible/roles/debops.rails_deploy/tasks/deploy_keys.yml

```diff
@@ -31,7 +36,12 @@
         not 'github' in rails_deploy_git_host
 
 - name: Transfer the deploy key to Gitlab
-  command: "curl --insecure --header 'PRIVATE-TOKEN: {{ rails_deploy_git_access_token }}' --data '{{ rails_deploy_key_data | to_nice_json }}' https://{{ rails_deploy_git_host }}/api/v3/projects/{{ rails_deploy_register_gitlab_response.json.id }}/keys"
+  command: "curl --insecure --header 'PRIVATE-TOKEN: {{ rails_deploy_git_access_token }}'
+                 --data '{{ rails_deploy_key_data | to_nice_json }}'
+                 https://{{ rails_deploy_git_host }}/api/v3/projects/{{
+                   rails_deploy_register_gitlab_response.json.id }}/keys"
+  args:
+    warn: False
```

Comment:
It should be possible to not depend on `curl` here and do it with a Ansible module. Also the `--insecure` gives me nightmares.

Reply:
Agreed, the `rails_deploy` role is unfortunately neglected, I'm not using it personally. Perhaps some kind of survey or poll to find what roles are popular among users could be interesting to do. Food for thought.

Reply:
Fair enough. Maybe a FIXME note can be added for now as this really needs fixing.